src/validation.cpp

```diff
@@ -5857,8 +5855,7 @@ bool ChainstateManager::PopulateAndValidateSnapshot(
     // ActivateSnapshot(), but is done so that we avoid doing the long work of staging
     // a snapshot that isn't actually usable.
     if (WITH_LOCK(::cs_main, return !CBlockIndexWorkComparator()(ActiveTip(), snapshot_start_block))) {
-        LogPrintf("[snapshot] activation failed - work does not exceed active chainstate\n");
-        return false;
+        return util::Error{Untranslated("Activation failed - work does not exceed active chainstate")};
```

Comment:
In commit "rpc: Return precise loadtxoutset error messages" (fa91404c68b4d503e6568f49aaf74a5746887185)

The error message seems inconsistent with other ones because it is prefixed with "Activation failed -". Maybe drop that prefix or also include it in the other messages?